README.md

```diff
@@ -36,25 +36,25 @@ The following reference pages are available to describe the Azure App Configurat
 
 ## Client Libraries
 
-### Configuration Providers
+#### Configuration Providers
 
 Module | Platform | Version | Sample | Release Notes
 ------ | -------- | ------- | ------ | -------------
-Microsoft.Extensions.Configuration.AzureAppConfiguration | .Net Standard | [![NuGet](https://img.shields.io/nuget/v/Microsoft.Extensions.Configuration.AzureAppConfiguration.svg?color=blue)](https://www.nuget.org/packages/Microsoft.Azure.AppConfiguration.AspNetCore/) | [Sample](https://github.com/Azure/AppConfiguration/tree/master/examples/DotNetCore) | [Release Notes](https://github.com/Azure/AppConfiguration/blob/master/releaseNotes/MicrosoftAzureAppConfigurationAspNetCore.md)
-Microsoft.Azure.AppConfiguration.AspNetCore | ASP&#46;NET Core | [![NuGet](https://img.shields.io/nuget/v/Microsoft.Azure.AppConfiguration.AspNetCore.svg?color=blue)](https://www.nuget.org/packages/Microsoft.Azure.AppConfiguration.AspNetCore/) | [Sample](https://github.com/Azure/AppConfiguration/tree/master/examples/DotNetCore) | [Release Notes](https://github.com/Azure/AppConfiguration/blob/master/releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md)
-[Microsoft.Configuration.ConfigurationBuilders.AzureAppConfiguration](https://github.com/aspnet/MicrosoftConfigurationBuilders/tree/master/src/AzureAppConfig) | .NET Framework | [![NuGet](https://img.shields.io/nuget/v/Microsoft.Configuration.ConfigurationBuilders.AzureAppConfiguration.svg?color=blue)](https://www.nuget.org/packages/Microsoft.Configuration.ConfigurationBuilders.AzureAppConfiguration/) |  | 
+Microsoft.Extensions.Configuration. AzureAppConfiguration | .Net Standard | [![NuGet](https://img.shields.io/nuget/v/Microsoft.Extensions.Configuration.AzureAppConfiguration.svg?color=blue)](https://www.nuget.org/packages/Microsoft.Azure.AppConfiguration.AspNetCore/) | [Sample](https://github.com/Azure/AppConfiguration/tree/master/examples/DotNetCore) | [Release Notes](https://github.com/Azure/AppConfiguration/blob/master/releaseNotes/MicrosoftAzureAppConfigurationAspNetCore.md)
+Microsoft.Azure.AppConfiguration.<br/>AspNetCore | ASP&#46;NET Core | [![NuGet](https://img.shields.io/nuget/v/Microsoft.Azure.AppConfiguration.AspNetCore.svg?color=blue)](https://www.nuget.org/packages/Microsoft.Azure.AppConfiguration.AspNetCore/) | [Sample](https://github.com/Azure/AppConfiguration/tree/master/examples/DotNetCore) | [Release Notes](https://github.com/Azure/AppConfiguration/blob/master/releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md)
```

Comment:
I'm not sure we should be adding custom line breaks for styling. The styling outcome is relative to screen size, trying to work around it may not work as expected.

Reply:
True. I hope there is a better way. But the current one looks very bad - the version is hardly readable. The fix didn't put any hard coded size. It only split the name into multiple lines. They will scale as normal.

Reply:
Alright. I removed the version column and put it to the same column of the module name. This avoided splitting the module name to multiple lines but still gave enough space for the version to be readable. Does this look better?

BTW, I also added a one-sentence description of what each section is about.

Reply:
This new way looks very good. It also is good usage of <br/>.